src/main/java/com/feedzai/commons/sql/abstraction/dml/result/PostgreSqlResultColumn.java

```diff
@@ -32,4 +34,19 @@
     public PostgreSqlResultColumn(final String name, final Object val) {
         super(name, val);
     }
+
+    /**
+     * Overrides default behaviour for JSON values, that are converted to strings.
+     *
+     * @param o The object in need of some kind of processing before being set.
+     * @return
+     */
+    @Override
+    protected Object processObject(Object o) {
+        if (o != null  && o instanceof PGobject &&
```

Comment:
null check is not needed, instanceof will return false if the left operand is null.